manager/integration/tests/common.py

```diff
@@ -2517,7 +2517,6 @@ def wait_pod_auto_attach_after_first_backup_completion(
                 completed = True
                 break
         if completed:
-            wait_for_volume_detached(client, vol.name)
```

Comment:
Correct. We should not wait and check a transient state.
But the naming of this function is confusing. In Longhorn, auto attachment typically means that the volume is attached by Longhorn itself. But here the volume attachment is requested by the workload.

Reply:
Renamed.